src/app/ClioApplication.cpp

```diff
@@ -32,8 +32,8 @@
 #include "rpc/common/impl/HandlerProvider.hpp"
 #include "util/Assert.hpp"
 #include "util/build/Build.hpp"
-#include "util/config/Config.hpp"
 #include "util/log/Logger.hpp"
+#include "util/newconfig/ConfigDefinition.hpp"
```

Comment:
Can we rename `newconfig` to `config` now?

Reply:
I was thinking after this PR is merged, ill create a new PR that deletes the old config and rename my `newconfig` to `config`.  so the PR isn't messy. WDYT?